avatars/avatars.js

```diff
@@ -1,4 +1,5 @@
 import * as THREE from 'three';
+import { SimplexNoise } from 'three/examples/jsm/math/SimplexNoise.js';
```

Comment:
We have a different noise in procgen.js: https://github.com/webaverse/app/blob/215519e150ffa94168d52e77f9cc8b489583bb89/procgen/procgen.js#L25

I don't care which one we use but other code uses this one so we should be consistent.

Reply:
used noise from simplex-noise.js